packages/react/src/components/Checkbox/index.tsx

```diff
@@ -92,6 +110,9 @@ const Checkbox = forwardRef<HTMLInputElement, CheckboxProps>(
                 onBlur(e);
               }
             }}
+            aria-checked={
```

Comment:
I'm not sure this is necessary. 

Looking at the accessibility tree, a checkbox with the value of `indeterminate` already has the right properties:

![chrome accessibility tree showing a value of checked: mixed](https://github.com/user-attachments/assets/8bc3002b-d628-464f-8c14-0d4ed068e4d8)

Reply:
Thanks, removed